### PR TITLE
CI tests for PostgreSQL

### PR DESCRIPTION
**Pull request by hugomiguelabreu**

### CODE REVIEW

File: .github/workflows/test-postgresql.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    - name: Cache Maven packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+
+    - name: Test with PostgreSQL
+      run: mvn -B clean test -Ppostgresql
```
Comment (artpdr) [inline .github/workflows/test-postgresql.yml:31]: Question: On average, how much time will tests take in each job?

Comment (hugomiguelabreu) [reply .github/workflows/test-postgresql.yml:31]: A few minutes. PostgreSQL is 3 minutes. But DB2 locally takes 10 minutes.

Comment (artpdr) [reply .github/workflows/test-postgresql.yml:31]: I was thinking in running tests for each profile but 10 minutes is probably too much

**Review by artpdr — APPROVED** [review]

File: .github/workflows/test-postgresql.yml (hunk 1 above)
Comment (hugomiguelabreu) [reply .github/workflows/test-postgresql.yml:31]: We will :) but each profile can go into a single workflow, this way they can run in parallel

**Pull request merged**